### PR TITLE
Allow RO servers to run without Kerberos enabled.

### PR DESCRIPTION
**Pull request by FelixGV**
In order to address Issue #278.

### CODE REVIEW

**Comment by singhsiddharth** [comment]
lgtm

**Comment by FelixGV** [comment]
Thanks, I merged it in.

File: contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java [hunk 1]
```diff
@@ -196,12 +196,17 @@ private Configuration getConfiguration(String sourceFileUrl, String hadoopConfig
 
             String security = config.get(CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION);
 
-            if(security == null || !security.equals("kerberos")) {
-                logger.error("Security isn't turned on in the conf: "
-                             + CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION + " = "
-                             + config.get(CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION));
-                logger.error("Please make sure that the Hadoop config directory path is valid.");
-                throw new VoldemortException("Error in getting Hadoop filesystem. Invalid Hadoop config directory path.");
+            if (security == null) {
```
Comment (arunthirupathi) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Nitpick : Can we make this a table based method and only warning for the simple ? This code kind of looks little awkward with check for simple and throws if not kerberos, which does the right thing, but little harder to read.

A code like valid options are simple and kerberos. If not throw.
If option equals simple, log a warning is easier to understand.

Comment (FelixGV) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Sure, let me re-jiggle that around a bit.

Comment (FelixGV) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Here's a new version of that block of code, which is a bit simpler: 10b2c76b55de9c38b8ace77f792784b4e0641fa7

LMK if that's okay for you.

Comment (FelixGV) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Easier to look at it this way: https://github.com/voldemort/voldemort/blob/10b2c76b55de9c38b8ace77f792784b4e0641fa7/contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java#L199

Comment (arunthirupathi) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Looks much better than what I suggested. Thanks for making this change.

Comment (FelixGV) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java]: Ok thanks, I merged it in.

**Pull request merged**